README

```diff
@@ -554,12 +554,12 @@ ScummVM will skip copy protection in the following games:
   * Zak McKracken and the Alien Mindbenders
 
 
-3.2) Datafiles:
+3.2) Data files:
 ---- ----------
-For a comprehensive list of required Datafiles for supported games
+For a comprehensive list of required Data files for supported games
 visit:
 
-  <http://wiki.scummvm.org/index.php/Datafiles>
+  <http://wiki.scummvm.org/index.php/Data files>
```

Comment:
You should not add a space here as it will break the URL link unless the Wiki is updated as well.

I am not sure about the Team POV, but personally I would leave this as "datafiles" in all cases as these kind of compound nouns are common in English and while I agree that "data file" is more orthographically accurate, the term "datafile" or "data-file" are at least as common in general usage... plus changing this in all cases could prove to be a huge headache for little improvement.